### PR TITLE
Add option to change ReleaseType

### PR DESCRIPTION
**Pull request by Dynakarp**
This PR adds a parameter to change the ReleaseType instead of hardcoding it as Production.

### CODE REVIEW

File: src/UUPDownload/CLIOptions.cs [hunk 1]
```diff
@@ -58,6 +58,9 @@ internal class DownloadRequestOptions
 
         [Option('l', "language", HelpText = @"The language to get. Must be used with the edition parameter. Omit either of these to download everything. Example: en-US", Required = false, Default = "")]
         public string Language { get; set; }
+
+        [Option('z', "releasetype", HelpText = @"The release type to report to the Windows Update servers. Example: Production, other example: Test", Required = false, Default = "Production")]
```
Comment (riverar) [inline src/UUPDownload/CLIOptions.cs:62]: We don't need more than one example, thanks though. Can you trim that help string?

**Review by riverar — CHANGES_REQUESTED** [review]

**Comment by riverar** [comment]
I was in the middle of merging this, are you done? :)

**Comment by Dynakarp** [comment]
Ah yeah, I'm done now.

**Comment by riverar** [comment]
Thanks @Dynakarp!

**Pull request merged**